mod-source-record-manager-server/src/main/java/org/folio/verticle/consumers/util/JournalParams.java

```diff
@@ -0,0 +1,158 @@
+package org.folio.verticle.consumers.util;
+
+import org.folio.DataImportEventPayload;
+import org.folio.rest.jaxrs.model.JournalRecord;
+
+import java.util.Arrays;
+import java.util.Map;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class JournalParams {
+
+  public JournalRecord.ActionType journalActionType;
+  public JournalRecord.EntityType journalEntityType;
+  public JournalRecord.ActionStatus journalActionStatus;
+
+  public JournalParams(JournalRecord.ActionType journalActionType,
+                       JournalRecord.EntityType journalEntityType,
+                       JournalRecord.ActionStatus journalActionStatus) {
+    this.journalActionType = journalActionType;
+    this.journalEntityType = journalEntityType;
+    this.journalActionStatus = journalActionStatus;
+  }
+
+  private interface IJournalParams<H> {
```

Comment:
could you pls remove the unused type parameter

Reply:
Fixed.